omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/DiagnosisResource2_2.java

```diff
@@ -98,6 +99,9 @@ public void purge(Diagnosis diagnosis, RequestContext requestContext) throws Res
 	public DelegatingResourceDescription getRepresentationDescription(Representation representation) {
 		if (representation instanceof DefaultRepresentation) {
 			DelegatingResourceDescription description = new DelegatingResourceDescription();
+			if (ModuleUtil.isOpenmrsVersionInVersions("2.5.* - 9.*")) {
+				description.addProperty("formNamespaceAndPath");
+			}
```

Comment:
@mseaton were you suggesting changes something like this?

Reply:
Something like this, yes.  Obviously you'd need to test this resource against version < 2.5 and versions >= 2.5 to ensure it works.  It would be nice if there were other convenience methods in ModuleUtil that allowed you to just pass in the minimum version, but if this is the best available, that's fine.  I'd at least create a convenience method in this class and use that (eg. `supportsFormNamespaceAndPath() { return ModuleUtil.isOpenmrsVersionInVersions("2.5.* - 9.*") }`

Reply:
Are there any convenience methods I could use to test this resource against versions >= 2.5?